### DISCUSSION TITLE
Competition evaluation Gazebo setup

### DISCUSSION THREAD

**Discussion by RPS98** [larics/icuas24_competition#41] [Q&A] [answered]
To develop the vision detector module, which one of the following Gazebo setups will be used during the evaluation, as it affects fruit detection?

![Gazebo_nogpu](https://github.com/larics/icuas24_competition/assets/57761265/cd617107-850c-45d8-9f58-fd8e57de8880)
![Gazebo_gpu](https://github.com/larics/icuas24_competition/assets/57761265/72bd07ac-b05a-406e-b29b-899c0c1ed060)

Thank you very much for the help.
Rafael

**Comment by fpetric** [comment, marked as answer]
I completely missed this one. It is the white setup without shadows (helps with the rendering speed)
